aws/resource_aws_ami.go

```diff
@@ -190,6 +198,18 @@ func resourceAwsAmi() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"platform_details": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"platform": {
```

Comment:
totally up to you if you wanna move this one up..already feels so much nicer with the order!

Reply:
nice catch! i guess a future effort